common/src/main/java/com/github/khanshoaib3/minecraft_access/MainClass.java

```diff
@@ -124,24 +127,27 @@ private static void _clientTickEventsMethod(MinecraftClient minecraftClient) {
 
         PositionNarrator.getInstance().update();
 
-        if (healthNHunger != null && otherConfigsMap.isHealthNHungerEnabled())
-            healthNHunger.update();
+        boolean worldAndPlayerExists = Objects.nonNull(MinecraftClient.getInstance()) && Objects.nonNull(WorldUtils.getClientPlayer());
+        if (worldAndPlayerExists) {
```

Comment:
`MinecraftClient.getInstance()` is never null, and thus checking for it is superfluous. This is also significantly clearer as a regular null check as opposed to using a variable.
```suggestion
        if (WorldUtils.getClientPlayer() != null) {
```

Reply:
I think if this mod is running on server, that instance is null.

Reply:
That would make sense, I kind of only looked at the constructor for `MinecraftClient` and saw that it was always set there forgetting that the constructor is only called from the client. I still feel that the additional nesting that `Objects.nonNull()` adds makes it significantly harder to read, the [Javadoc](https://docs.oracle.com/javase/8/docs/api/java/util/Objects.html#nonNull-java.lang.Object-) for it states that it exists for use as a predicate for methods such as `Stream.filter()`. Using a variable for this condition is also superfluous because a regular null check is usually easy enough to read.
```suggestion
        if (MinecraftClient.getInstance() != null && WorldUtils.getClientPlayer() != null) {
```

Reply:
Yep, you're right, will accept it.